docs/subnets/walkthrough-prompting.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Walkthrough of Subnet 1"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Walkthrough of Subnet 1
+
+In this section we present a high-level walkthrough of the [Subnet 1: Prompting](https://github.com/opentensor/prompting/tree/main). This subnet creates  human-conversation-like AI for a variety of tasks such as:
+
+- Answering questions.
+- Summarizing a given text.
+- Debugging code. 
+- Solve mathematics problems, and more.
+
+:::tip Explorer
+You can see the prompting subnet in action on the [Taostats explorer (select Subnet 01: Text Prompting from the top menu)](https://taostats.io/). 
+:::
+
+## Before you proceed
+
+If you are new to Bittensor subnets and building blocks, read the following sections before you proceed further:
+
+- [Bittensor Building Blocks](../learn/bittensor-building-blocks).
+- [Anatomy of Incentive Mechanism](../learn/anatomy-of-incentive-mechanism).
+
+The below diagram shows a typical subnet with many subnet miners and subnet validators together executing the subnet incentive mechanism code. On the [Taostats explorer (select Subnet 01: Text Prompting from the top menu)](https://taostats.io/) the **Metagraph** view for this Subnet 1: Prompting shows the performance details for each subnet miner and validator. 
+
+For easier understanding, in this document we will focus on how a **single** subnet validator interacts with **multiple** subnet miners in this subnet. 
+
+<center id="bittensor-img">
+<ThemedImage
+alt="1-Prompting Walkthrough"
+sources={{
+    light: useBaseUrl('/img/docs/1-prompting-subnet-walkthrough.svg'),
+    dark: useBaseUrl('/img/docs/1-prompting-subnet-walkthrough.svg'),
+  }}
+style={{width: 600}}
+/>
+</center>
+
+## Subnet 1 summary
+
+See the below diagram showing a high-level view of how this Prompting subnet works. 
+
+<center>
+<ThemedImage
+alt="Prompting Subnet 1 Big Picture"
+sources={{
+    light: useBaseUrl('/img/docs/2-prompting-subnet-high-level.svg'),
+    dark: useBaseUrl('/img/docs/2-prompting-subnet-high-level.svg'),
+  }}
+style={{width: 750}}
+/>
+</center>
+
+The numbered items in the below description correspond to the numbered sections in the above diagram:
+
+1. The subnet validator sends a **challenge** simultaneously to multiple subnet miners. This step constitutes the **query** from the subnet validator to subnet miners. A challenge is a prompt sent to subnet miners in such a way that:
```

Comment:
Feels like we're sweeping quite a lot under the rug with this high level. My opinion is we could dip our toes a little further into the technical (or at least novel) aspects of the subnet. 

For instance, to prevent miners from simply 'looking up' the answers we have developed a novel approach which introduces 'fuzziness' and requires the use of semantic intelligence to understand instructions.